frontend/app/(defaultLayout)/Troubleshooting/[device]/components/Answers.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, Heading, Link, SimpleGrid, Stack, Text } from '@chakra-ui/react';
+import { TroubleshootingAnswersData } from '../hooks/useTroubleshootingProblemsProps';
+import { AnswerCard } from './AnswerCard';
+
+export function Answers({
+   answers,
+   allAnswersUrl,
+   allAnswersCount,
+}: TroubleshootingAnswersData) {
```

Comment:
Can't we just use count of `answers` instead of passing in another param?

Reply:
We limit the API data to return 10 entires, so that won't in this situation.